src/lib/openapi/spec/bulk-registration-schema.ts

```diff
@@ -20,29 +22,48 @@ export const bulkRegistrationSchema = {
                     },
                 },
             },
+            example: ['unleash-edge'],
         },
         appName: {
+            description:
+                'The name of the application that is evaluating toggles',
             type: 'string',
+            example: 'Ingress load balancer',
         },
         environment: {
+            description: 'Which environment is the application running in',
             type: 'string',
+            example: 'development',
         },
         instanceId: {
+            description: 'A unique identifier for the application',
             type: 'string',
+            example: 'Random UUID',
         },
         interval: {
+            description:
+                'How often (in seconds) does the application refresh its features',
             type: 'number',
+            example: 10,
         },
         started: {
+            description: 'When did the application start',
+            example: '1952-03-11T12:00:00.000Z', //GNU Douglas Adams
             $ref: '#/components/schemas/dateSchema',
         },
         strategies: {
+            description:
+                'Which [strategies](https://docs.getunleash.io/reference/activation-strategies) are enabled in the application',
             type: 'array',
+            example: ['standard', 'gradualRollout'],
             items: {
                 type: 'string',
             },
         },
         sdkVersion: {
+            summary: 'Version identifier for the SDK used',
+            description: 'Typically <client>:<version>',
```

Comment:
Properties don't have a "summary" field. It's all description. How about this?

```suggestion
            summary: 'Version identifier for the SDK used. Typically in the format of "<client>:<version>"',
```